superset-frontend/packages/superset-ui-demo/storybook/stories/superset-ui-time-format/TimeFormatStories.tsx

```diff
@@ -17,30 +17,25 @@
  * under the License.
  */
 
-/* eslint-disable jsx-a11y/label-has-associated-control */
 import React from 'react';
 import { formatTime } from '@superset-ui/core';
 
-const propTypes = {};
-const defaultProps = {};
-
 class TimeFormatValidator extends React.PureComponent {
+  state: { formatString: string; testValues: any[] } = {
```

Comment:
same as above.
```suggestion
  state: { formatString: string; testValues: (Date | number | null | undefined)[] } = {
```

Reply:
This typing do suggest another change in the file
`superset-frontend/packages/superset-ui-core/src/time-format/TimeFormatterRegistrySingleton.ts::formatTime() method`

`value: Date | number | null | undefined, --> value: Date | null | undefined,` Hope it is okay to change this. Functionally it seem to work. So making that change also.